centreon/packages/ui/src/Graph/Chart/Legend/Legend.styles.ts

```diff
@@ -1,17 +1,33 @@
 import { makeStyles } from 'tss-react/mui';
 
+import { equals, lt } from 'ramda';
 import { margin } from '../common';
+import type { LegendModel } from '../models';
 
 interface MakeStylesProps {
   limitLegendRows?: boolean;
+  placement: Pick<LegendModel, 'placement'>;
+  height?: number | null;
 }
 
 export const legendWidth = 21;
 const legendItemHeight = 5.25;
 const legendItemHeightCompact = 2;
 
+const getLegendMaxHeight = ({ placement, height }) => {
+  if (!equals(placement, 'bottom')) {
+    return height || 0;
+  }
+
+  if (lt(height || 0, 220)) {
+    return 40;
+  }
+
+  return 90;
```

Comment:
Where the numbers come from?

Reply:
that from original code , i assumed these numbers were set by ux Team